### DISCUSSION TITLE
Sorry  to an pain, no issue but can't find the command

### DISCUSSION THREAD

**Discussion by npaisnel** [HeyuX10Automation/heyu#62] [Q&A] [answered]
Sorry, Hope someone will be able to put me straight here.

I am sure I used to used a issue a command at the ssh or local terminal to view the uploaded schedule of timers 

something like "heyu show ???"

But I am damed if i can find the command,  tried x10.sched and conf man pages and heyu man pages but just can't see it.

it would bring up a 'table' from what I remember 

Thanks

**Comment by heyu-users** [comment, marked as answer]
Perhaps you're thinking of "heyu upload check"

This generates a file with a detailed report on the schedule.

Tom Henderson

**Comment by npaisnel** [comment]
Ah yes, sort of .  

That report file does contain the table I was thinking of, but I had a memory that it could be displayed directly from the terminal on the local screen without using nano to read the report.txt file.

Reading the report.txt does give me the information I wanted though, thanks
